packages/extension-chains/src/chains.ts

```diff
@@ -2,15 +2,14 @@
 // SPDX-License-Identifier: Apache-2.0
 
 import { MetadataDefBase } from '@polkadot/extension-inject/types';
-import { all } from '@polkadot/networks';
+import networks from '@polkadot/networks';
 
 const hashes: MetadataDefBase[] =
-all.filter(({ genesisHash }) => !!genesisHash)
-  .map((network) => ({
-    chain: network.displayName.replace('Relay Chain', ''),
-    genesisHash: (network.genesisHash as string[])[0],
-    icon: network.icon || 'jdenticon',
-    ss58Format: network.prefix
-  }));
+networks.map((network) => ({
+  chain: network.displayName,
+  genesisHash: network.genesisHash?.[0] || '0x',
```

Comment:
With the current version (.17 I believe) genesisHash is always defined. (However, it may be an empty array, so `network.genesisHash[0] || '0x')

Reply:
I've the following type with the latest (this one)
```
export interface Network extends NetworkFromSubstrate {
    genesisHash: string[] | null;
    icon: Icon;
}
```

Reply:
https://github.com/polkadot-js/common/blob/master/packages/networks/src/types.ts#L18

Reply:
The only one in that list that won't have a genesisHash is 42, which is the default prefix (but not really a chain)

Reply:
ah didn't think we'd use beta for this package. Pulling that one :)

Reply:
> The only one in that list that won't have a genesisHash is 42, which is the default prefix (but not really a chain

yeah I figured out, I've put "0x" here to keep the ordering because en empty string would have put the generic accounts at the bottom, which is not what we want.